.github/workflows/qemu-kvm-integration-tests.yml

```diff
@@ -164,6 +186,32 @@ jobs:
           done
           exit $rc
 
+      - name: Run bootc validation tests in QEMU
+        if: steps.check_platform.outputs.supported &&
+            startsWith(matrix.scenario.env, 'container') &&
+            endsWith(matrix.scenario.image, '-bootc')
+        run: |
+          set -euxo pipefail
+          env=$(echo "${{ matrix.scenario.env }}" | sed 's/^container-/qemu-/')
+
+          for t in $(grep -wl tests::bootc-e2e tests/tests_*.yml); do
+              test=$(basename ${t%.yml})
+              image_file="$(pwd)/tests/tmp/$test/qcow2/disk.qcow2"
```

Comment:
Ah - this is where the `/tmp/` directory is used.  Hmm - not sure how to get the `$LSR_TOX_ENV_TMP_DIR` in this context - this may also be problematic when testing the collections format, because the test directory is `tests/$rolename/` and not `....$rolename/tests/` there

Reply:
This isn't `$LSR_TOX_ENV_TMP_DIR` at the moment, but `OUTPUT="./tmp/$BUILDAH_ID"` in bootc-buildah-qcow.sh (in the tox-lsr PR). We can of course change it on both sides.